deviceconnect/connect.go

```diff
@@ -57,3 +63,12 @@ func Connect(serverUrl string, connectUrl string, token string) (ws *websocket.C
 	})
 	return ws, nil
 }
+
+func getWebSocketScheme(scheme string) string {
+	if scheme == httpsProtocol {
+		scheme = wssProtocol
+	} else if scheme == httpProtocol {
+		scheme = wsProtocol
+	}
+	return scheme
+}
```

Comment:
right! I thought that we just enforce ws:// or wss:// in config and we do not allow http:// or https:// but there is no harm in merging that.